FunctionSignatureMap.php

```diff
@@ -15152,7 +15160,8 @@
 'VarnishLog::getTagName' => ['string', 'index'=>'int'],
 'VarnishStat::__construct' => ['void', 'args='=>'array'],
 'VarnishStat::getSnapshot' => ['array'],
-'version_compare' => ['int|bool', 'ver1'=>'string', 'ver2'=>'string', 'oper='=>'\'\x3c\'|\'lt\'|\'\x3c=\'|\'le\'|\'\x3e\'|\'gt\'|\'\x3e=\'|\'ge\'|\'==\'|\'=\'|\'eq\'|\'!=\'|\'\x3c\x3e\'|\'ne\''],
+'version_compare' => ['bool', 'ver1'=>'string', 'ver2'=>'string', 'oper'=>'\'\x3c\'|\'lt\'|\'\x3c=\'|\'le\'|\'\x3e\'|\'gt\'|\'\x3e=\'|\'ge\'|\'==\'|\'=\'|\'eq\'|\'!=\'|\'\x3c\x3e\'|\'ne\''],
+'version_compare\'1' => ['int', 'ver1'=>'string', 'ver2'=>'string', 'oper='=>''],
```

Comment:
Aside: The third oper should probably be removed from the alternate signature in psalm.

```
php > var_export(version_compare(1, 2, null));
true
php > var_export(version_compare(1, 2, false));
true
php > var_export(version_compare(1, 2, ''));
true
```

Phan currently does a poor job of pattern matching alternate signatures (in the backlog), which is why many of these signatures are combined in Phan's version